test/org/loklak/harvester/RedirectUnshortenerTest.java

```diff
@@ -21,7 +21,7 @@ public void testRedirectUnshortener() {
         shortlinkMap.put("http://ow.ly/P9SX30ddSTI", "https://github.com/loklak/loklak_server/issues/1284");
         shortlinkMap.put("http://bit.do/blog-fossasia", "http://blog.fossasia.org/");
         shortlinkMap.put("http://fb.me/4lcXZsyyO", "https://www.facebook.com/permalink.php?story_fbid=1550262581900846&id=1381813572079082");
-        shortlinkMap.put("http://wp.me/sf2B5-shorten", "https://en.blog.wordpress.com/2009/08/14/shorten/");
+        shortlinkMap.put("http://wp.me/sf2B5-shorten", "http://en.blog.wordpress.com/2009/08/14/shorten/");
```

Comment:
@hemantjadon: I never mentioned that `http` or `https` has anything to do with anonymity. I was just saying that we do not need to resolve a URL from `http` to `https` if it isn't from any url shortening service (which generally maintain stats about who is visiting the link, from where, interests, etc.).